RecoEgamma/EgammaElectronAlgos/interface/getCtfTrackRef.h

```diff
@@ -0,0 +1,14 @@
+#ifndef getCtfTrackRef_h
```

Comment:
Include guard should include the full path

Reply:
Ok! Thanks for giving your opinion.